modules/rdg/src/userobjects/SlopeLimitingBase.C

```diff
@@ -52,14 +52,6 @@ SlopeLimitingBase::initialize()
   _lslope.clear();
 }
 
-void
-SlopeLimitingBase::threadJoin(const UserObject & y)
-{
-  const SlopeLimitingBase & pps = static_cast<const SlopeLimitingBase &>(y);
-
-  _lslope.insert(pps._lslope.begin(), pps._lslope.end());
```

Comment:
I believe so, I removed a few that Derek put in there last year on GeneralUserObjects.